### ISSUE TITLE
Pause/Resume with Views

### ISSUE THREAD

**Issue by peervm** [devcartel/pyrfa#32]
Hi,

Does the pause/resume feature work with views?

I used the examples/view.py code and added pause/resume. Unfortunately after _resume_ was called then I started to receive all updates and not just my filtered view.

Thanks,
Peer

**Comment by wiwat-tharateeraparb** [comment]
I think it’s a bug. When resuming, it doesn’t pass the set views to the server. I will fix this in the next release. ETA is about 15 days.

**Comment by peervm** [comment]
Awesome, thanks for the prompt reply and future fix.

Thanks,
Peer

**Comment by wiwat-tharateeraparb** [comment]
Seems that `pauseAll()` and `resumeAll()` work with views but not `marketPriceResume(ric)`

**Comment by peervm** [comment]
Confirmed that views combined with 'pauseAll/resumeAll' behaves as
expected. Thanks.

**Comment by wiwat-tharateeraparb** [comment]
Please try PyRFA 8.2.1 released today for which addresses this issue.

**Comment by peervm** [comment]
Great, thanks for the quick fix. I`ll try it out this friday. Thanks.

**Comment by peervm** [comment]
Hi, I tried PyRFA 8.2.1 using `marketPricePause(ric)` and `marketPriceResume(ric)`  but it still loses the view settings after the resume. Other than checking the __init__.py version number how can I programmatically check the pyrfa.pyd version? I want to ensure I am running the correct version.

Also, I am using Python 3.6 win64. Thanks.

**Comment by wiwat-tharateeraparb** [comment]
You can run below to check the version:
```
>>> import pyrfa
>>> pyrfa.__version__
```
I will double check it

**Comment by wiwat-tharateeraparb** [comment]
I have checked again with PyRFA 8.2.1 Win64 and can confirm that view works with `marketPricePause` and `marketPriceResume`. Maybe you can try to update PyRFA with:

```
Python3.6.exe -m pip install -U pyrfa
```

**Comment by wiwat-tharateeraparb** [comment]
Fixed in 8.2.1
